### DISCUSSION TITLE
Team based access for Application or services

### DISCUSSION THREAD

**Discussion by iamdineshkumar** [coollabsio/coolify#456] [Q&A] [answered]
I have gone through team module but there is no clarity what team does. Based on my understanding it should be available to give access to environment/container spinned to team. But i am not able to find such option so what team does ?

**Comment by andrasbacsai** [comment]
Each resource (application, database, services, destination, sources) is assigned to a team. Each team has its members. 

So it can be used to separate resources between teams/users.

**Reply by iamdineshkumar** [reply]
But there is no flag/label to understand which application/services under which team. I know that user can create multiple teams in that case he also should have rights to assign application to teams based on necessity right?

**Reply by peaklabs-dev** [reply, marked as answer]
Resources are created inside a specify team in v4
